frontend/utils/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './breakpoints'
+export * from './colors'
+export * from './css'
+export * from './date'
+
+// Polyfill Object.fromEntries
```

Comment:
I was having trouble with next.js / TypeScript allowing me to use es2019 `Object.fromEntries` so I just polyfilled it